src/main/java/com/zaxxer/hikari/pool/PoolEntry.java

```diff
@@ -96,9 +108,21 @@ public void setKeepalive(ScheduledFuture<?> keepalive) {
       this.keepalive = keepalive;
    }
 
-   Connection createProxyConnection(final ProxyLeakTask leakTask)
-   {
```

Comment:
Do not change the formatting of the code if there is not change in the line.  In original formatting "{" was placed in new line.